en/docs/install-and-setup/configuring-api-manager-for-ob.md

```diff
@@ -76,8 +76,8 @@ database server, and the JDBC driver.
     ```
 
 7. `FrequencyPerDay` is a header parameter sent in the API invocation request to indicate the requested maximum 
```

Comment:
Shall we remove the part which says "FrequencyPerDay" is a header parameter? There is no header parameter called "FrequencyPerDay" in the specification. The frequencyPerDay value is sent with the consent initiation request body. We enforce the throttling only if the [open_banking_berlin.consent.freq_per_day] config is enabled and the throttling is correctly configured.

Reply:
Fixed via c682c8228800a36220143c903b5837285c404c0b.

Reply:
Created this issue to update OB200 and OB150 docs: https://github.com/wso2/docs-open-banking/issues/641